demo/admin/package.json

```diff
@@ -13,7 +13,7 @@
         "intl:compile:comet": "formatjs compile-folder --format simple --ast lang/comet-lang lang-compiled/comet-lang",
         "intl:compile:comet-demo": "formatjs compile-folder --format simple --ast lang/comet-demo-lang/admin lang-compiled/comet-demo-lang-admin",
         "intl:extract": "formatjs extract \"src/**/*.{ts,tsx}\" --ignore ./**.d.ts --out-file lang-extracted/en.json --format simple",
-        "lint": "run-s intl:compile && run-p gql:types generate-block-types && run-p lint:prettier lint:eslint lint:tsc && $npm_execpath lint:generated-files-not-modified",
+        "lint": "run-p intl:compile generate-block-types && $npm_execpath lint:generated-files-not-modified && run-p lint:prettier lint:eslint lint:tsc",
```

Comment:
Why this change?

Reply:
`gql:types` is not explicit needed, it's done in admin-generator command, before and after the call - like explained in the description.

That's why in `lint` script, the `intl:compile` and `generate-block-types` could run together in parallel -> faster.

`generate-files-not-modified` is internally using `adming-generator` and can run for it's own because it generates the `gql:types` on its own.

If we want to specially lint if `gql:types` succeeds, we probably could add it to the last part, where `lint:prettier`, `lint: eslint`, `lint:tsc` run. But if `gql:types` will fail anyways, it will never reach the last part because it will fail earlier in the lint chain.

Reply:
> generate-files-not-modified is internally using adming-generator and can run for it's own because it generates the gql:types on its own.

It's odd that we need to run the `gql:types` script before and after the generator. Won't this slow down the generator even more?

Reply:
the gql:types call before is not needed anymore (i think because of the discusses ts-morph change)